v3/src/components/graph/components/graph-axis.tsx

```diff
@@ -34,7 +34,8 @@ export const GraphAxis = observer(function GraphAxis(
     instanceId = useInstanceIdContext(),
     layout = useGraphLayoutContext(),
     droppableId = `${instanceId}-${place}-axis-drop`,
-    hintString = useDropHintString({role: axisPlaceToAttrRole[place]})
+    hintString = useDropHintString({role: axisPlaceToAttrRole[place]}),
+    getCategorySet = () => dataConfig?.categorySetForPlace(place)
```

Comment:
Is there reason to believe this needs to be a getter? Passing the categorySet instance directly would be simpler.

Reply:
Even simpler is to realize it was no longer being used and therefore can be eliminated entirely!